server/server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+let app = express();
+let port = 3123;
+const bodyParser = require('body-parser')
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(express.static(__dirname + '/../public'));
```

Comment:
This is a more semantic point, but express.static inherently uses path.resolve. It is unnecessary to use __dirname. It would be okay to not change this.

https://stackoverflow.com/questions/35799699/what-is-the-rule-for-express-static-root-path